<launch>
	<!-- DATA SERVICE-->
	<node 
		pkg="data_service"
		type="service.py"
		name="data_service"
	>
	</node>
	
	<!-- IMU NODE -->
	<node 
		pkg="barc"
		type="imu_data_acq.py"
		name="imu_node"
	>
		<param name="port" value="/dev/ttyACM0" />
	</node>

	<!-- ARDUINO NODE -->
	<node 
		pkg="rosserial_python"
		type="serial_node.py"
		name="arduino_node"
	>
		<param name="port" value="/dev/ttyUSB0" />	
	</node>

	<!-- AUTONOMOUS DRIVE NODE -->
	<!-- Test Selection
	0 : Circular 
	1 : Straight 
	2 : Sine Sweep
	3 : Double Lane Change
	4 : LQR Drfit
	--> 
	<node 
		pkg="barc"
		type="auto_mode.py"
		name="auto_node"
	>
		<param name="user" value="jgon13" />
		<param name="experiment_sel" type="int" value="0" />	

		<param name="v_x_pwm" type="int" value="98" />	
		<param name="steering_angle" type="int" value="10" />
		<param name="t_exp" type="int" value="15" />
	</node>
</launch>
